### PR TITLE
update to 14.0.6

### PR DESCRIPTION
**Pull request by katietz**

### CODE REVIEW

**Comment by ELundby45** [comment]
@katietz I believe we will need to add packages for both [p7zip](https://github.com/conda-forge/p7zip-feedstock) and [msitools](https://github.com/conda-forge/msitools-feedstock/) for the `linux-64` arch.

```
Unsatisfiable dependencies for platform linux-64: {'p7zip', 'msitools'}
```

Furthermore, `msitools` relies on [gcab](https://github.com/conda-forge/gcab-feedstock) and [libgsf](https://github.com/conda-forge/libgsf-feedstock) which it looks like we do not have.

**Comment by katietz** [comment]
Yes, if you want we can do this today together

File: recipe/meta.yaml [hunk 1]
```diff
@@ -41,6 +40,7 @@ outputs:
   - name: clang_win-64
     script: install-pkg.sh   # [not win]
     script: install-pkg.bat  # [win]
+    skip: True [not win]
```
Comment (marcoesters) [inline recipe/meta.yaml]:
```suggestion
    skip: True  # [not win]
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -41,6 +41,7 @@ outputs:
   - name: clang_win-64
     script: install-pkg.sh   # [not win]
     script: install-pkg.bat  # [win]
+    skip: True  # [not win]
```
Comment (marcoesters) [inline recipe/meta.yaml:44]: If we skip everything but `win` here, do we still need any of the `not win` lines and all the selectors in this output?

**Review by sumit0190 — APPROVED** [review]
LGTM; thanks

File: recipe/meta.yaml (hunk 2 above)
Comment (katietz) [reply recipe/meta.yaml:44]: yes, that is an conda-build issue ... it is adviced to specify a separete build-script, if a selector is used for it.

File: recipe/meta.yaml (hunk 1 above)
Comment (katietz) [reply recipe/meta.yaml]: thanks

**Pull request merged**